File: python/q7.py
#!/usr/bin/python3

def isPrime(n):
    for i in range(2,int(n**0.5)+1):
        if n%i==0:
            return False
    return True
counter=2
num=4
while counter<10001:
    if isPrime(num):
        counter += 1
    num+=1
print(num-1)

File: go/q7.go
package main
import("fmt"; "github.com/daniel-willett/common-go-methods")

func main(){
	var counter int = 2
	var num int = 4
	for counter<10001{
		if common.IsPrime(num)==true{
			counter += 1
		}
		num += 1
	}
	fmt.Println(num-1)
}

